scripts/devaccess/devaccess_users.yaml

```diff
@@ -0,0 +1,14 @@
+roles:
+- name: ALL
+  commands:
+  - oc get nodes -ndefault
+  - oc get nodes -ndefault -ojson
+  - rpm -qa
+- name: super_role
+  commands:
+  - uname
+
+users:
+- username: superperson
```

Comment:
After further review, I see this is a sample config, and won't be used as a live/initial config.  Granting priv escalation to root seems incorrect.  Perhaps "nobody" or "guest" would be better example users?